extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueLightHandler.java

```diff
@@ -125,6 +130,11 @@ private void initializeThing(@Nullable ThingStatus bridgeStatus) {
             if (getHueClient() != null) {
                 if (bridgeStatus == ThingStatus.ONLINE) {
                     initializeProperties();
+
+                    if (colorModeChannelRequired()) {
```

Comment:
What is the purpose of this logic? From the xml file I can tell that the 0210 thing has this channel, so why do you have this check and addition logic here?